Makefile.am

```diff
@@ -26,32 +26,7 @@ AUTOMAKE_OPTIONS = foreign
 
 ACLOCAL_AMFLAGS = -I m4
 
-CMAKE_DIST =                                    \
- CMake/cmake_uninstall.cmake.in                 \
- CMake/CMakeConfigurableFile.in                 \
- CMake/curl-config.cmake.in                     \
- CMake/CurlSymbolHiding.cmake                   \
- CMake/CurlTests.c                              \
- CMake/FindBearSSL.cmake                        \
- CMake/FindBrotli.cmake                         \
- CMake/FindCARES.cmake                          \
- CMake/FindGSS.cmake                            \
- CMake/FindLibPSL.cmake                         \
- CMake/FindLibSSH2.cmake                        \
- CMake/FindMbedTLS.cmake                        \
- CMake/FindMSH3.cmake                           \
- CMake/FindNGHTTP2.cmake                        \
- CMake/FindNGHTTP3.cmake                        \
- CMake/FindNGTCP2.cmake                         \
- CMake/FindQUICHE.cmake                         \
- CMake/FindWolfSSL.cmake                        \
- CMake/FindZstd.cmake                           \
- CMake/Macros.cmake                             \
- CMake/OtherTests.cmake                         \
- CMake/PickyWarnings.cmake                      \
- CMake/Platforms/WindowsCache.cmake             \
- CMake/Utilities.cmake                          \
- CMakeLists.txt
+CMAKE_DIST = CMakeLists.txt
```

Comment:
We do use `.inc` extension for other make include files, maybe that would make sense here too?

Reply:
Ah no sorry, it's not an include file. Strike this!

Reply:
There are other problems. I'm adjusting back not to add an extra automake directory (I don't know what I'm doing and it doesn't work).